server.js

```diff
@@ -442,22 +438,9 @@ function sessionCheck(req, res, next) {
 
 let dbAbstraction = new DbAbstraction();
 dbAbstraction.hello();
-async function dbPing() {
```

Comment:
Since you got rid of this function, there is no use of `isDbConnected` variable above. Please cleanup that.

And also, when the new client socket connection comes, make sure to give the right dbState to the client. For your reference - line number 272.